mode/htmlmixed/htmlmixed.js

```diff
@@ -74,7 +74,8 @@
       name: "xml",
       htmlMode: true,
       multilineTagIndentFactor: parserConfig.multilineTagIndentFactor,
-      multilineTagIndentPastTag: parserConfig.multilineTagIndentPastTag
+      multilineTagIndentPastTag: parserConfig.multilineTagIndentPastTag,
+      allowMissingTagName: parserConfig.allowMissingTagName,
```

Comment:
It doesn't look like this is used at all.

Reply:
This is passed through to xml (https://github.com/codemirror/CodeMirror/blob/master/mode/xml/xml.js#L55)

Reply:
Ah, right, my bad.